test/integration/echo_integration_test.cc

```diff
@@ -3,19 +3,36 @@
 #include "test/server/utility.h"
 
 namespace Envoy {
+
+std::string echo_config;
+
 class EchoIntegrationTest : public BaseIntegrationTest,
                             public testing::TestWithParam<Network::Address::IpVersion> {
 public:
-  EchoIntegrationTest() : BaseIntegrationTest(GetParam()) {}
+  EchoIntegrationTest() : BaseIntegrationTest(GetParam(), echo_config) {}
+
+  // Called once by the gtest framework before any EchoIntegrationTests are run.
+  static void SetUpTestCase() {
+    echo_config = ConfigHelper::BASE_CONFIG + R"EOF(
+    filter_chains:
+      filters:
+        name: envoy.ratelimit
+        config:
+          domain: foo
+          stats_prefix: name
+          descriptors: [{"key": "foo", "value": "bar"}]
+      filters:
+        name: envoy.echo
+        config:
+      )EOF";
+  }
+
   /**
-   * Initializer for an individual test.
+   *  Initializer for an individual test.
```

Comment:
errant ' ' ?

Reply:
I loves me my whitespace :-P

(no, this is flat out wrong, fixed locally)